### DISCUSSION TITLE
AttributeError: 'ROM' object has no attribute 'printXML' - when trying to print sobol data

### DISCUSSION THREAD

**Discussion by ortegapr** [idaholab/raven#2209] [Q&A]
Hello I am not able to print the sobol indices of my HDMRRom as an outstream, I think I am following the indications of the manual , could you please look at my code below?
`<?xml version="1.0" ?>
<Simulation color="true">
  
  <RunInfo>
    <JobName>ROM_PICKLE_SAMPLE</JobName>
    <WorkingDir>Sobol_Sampling2</WorkingDir>
    <Sequence> pk3Load, Print</Sequence>
    <batchSize>1</batchSize>
  </RunInfo>
  
  <Models>
    <ROM name='myHDMR' subType='HDMRRom'>
      <Features>l, T, H0, Cl, S1max, S0, t_final, Phi, Theta, FlawDepth, Lp, t_crack, AGS_Delay, wo, do</Features>
      <Target>Heq_TSSD, t_more_TSSD, Heq, LBB, first_2c, first_CCL, Time_2c, Q_2c</Target>
      <SobolOrder>2</SobolOrder>
      <PolynomialOrder>4</PolynomialOrder>
      <IndexSet>TotalDegree</IndexSet>
    </ROM>
  </Models>

  <Files>
    <Input name="HDMR" type="">HDMR.pk</Input>
  </Files>

  <Steps>
   <IOStep name='pk3Load'>
     <Input class='Files' type='' >HDMR</Input>
     <Output class='Models' type='ROM'>myHDMR</Output>

   </IOStep>
   <IOStep name="Print" pauseAtEnd="False">
      <Input  class="Models" type="ROM">myHDMR</Input>
      <Output class="OutStreams"  type="Print">SobolData</Output>
   </IOStep>
  </Steps>
  
  <OutStreams>
    <Print name="SobolData">
      <type>xml</type>
      <source>myHDMR</source>
      <what>all</what>
    </Print>
  </OutStreams>
  
</Simulation>    `

**Comment by wangcj05** [comment]
Sorry for the late response, we did some experimental testing using "Discussions", but we have not use it to track any issues. Please feel free to send us an email, or create a regular issue. In addition, if you still face the issue, could you attach the full raven xml input for us to take a look?
